File: Solutions/1963/1963.py
class Solution(object):
    def minSwaps(self, s):
        """
        :type s: str
        :rtype: int
        """
        x = 0
        for i in s:
            if i == '[':
                x += 1
            elif x > 0:
                x -= 1
        return (x + 1) // 2

File: Solutions/1963/1963.cs
public class Solution {
    public int MinSwaps(string s) {
        int x = 0;
        for (int i = 0; i < s.Length; i++)
        {
            if (s[i] == '[')
                x++;
            else if (x > 0)
                x--;
        }
        return (int) (x+1)/2;
    }
}
